exercises/practice/dnd-character/dnd_character.rb

```diff
@@ -0,0 +1 @@
+
```

Comment:
We are giving an empty file.  There is commonly a comment here that describes (for those that may have been away for a while) to write the tests in this file, and to read the Instructions or README if they are developing locally.

Reply:
Added boilerplate language for the stub file!